spec/features/admin_adds_new_video_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'spec_helper'
+
```

Comment:
You look to be using Rspec 3.1. You should be using the correct syntax for [feature specs](https://www.relishapp.com/rspec/rspec-rails/v/3-1/docs/feature-specs/feature-spec)